learn/appchains/overview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Appchain Overview
+description: Learn the high-level definitions of how a Tanssi Appchain works, its architecture, and the block production as a service mechanism with deterministic finality
```

Comment:
```suggestion
description: Learn the high-level definitions of how a Tanssi Appchain works, its architecture, and its block production as a service mechanism with deterministic finality.
```

Reply:
If this should remain as "the" that's cool, the main thing is the period at the end 🙂